src/main/scala/org/broadinstitute/dsde/workbench/avram/api/CollectionsServlet.scala

```diff
@@ -0,0 +1,72 @@
+package org.broadinstitute.dsde.workbench.avram.api
+
+import io.circe.generic.auto._
+import io.circe.syntax._
+import java.util.UUID
+import java.util.logging.Logger
+import javax.servlet.http.{HttpServlet, HttpServletRequest, HttpServletResponse}
+
+import org.broadinstitute.dsde.workbench.avram.model.{AvramException, SamResource}
+import org.broadinstitute.dsde.workbench.avram.service.CollectionsService
+
+import scala.concurrent.Future
+import scala.util.{Failure, Success}
+
+
+class CollectionsServlet extends HttpServlet with AvramServlet {
+
+  private val log = Logger.getLogger(getClass.getName)
+  private val collectionsService = new CollectionsService()
+
+  override def doGet(request: HttpServletRequest, response: HttpServletResponse) = {
```

Comment:
Before we get too far, I'd be interested to see if we can use a JAX-RS framework like [Jersey](https://jersey.github.io/). That should give us a nice way of specifying endpoints and extracting parameters from the path and maybe even serialization/deserialization (not sure if we'd be able to have it use circe).